SonarQube.Scanner.End/SonarQube.Scanner.End.csproj

```diff
@@ -0,0 +1,57 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AssemblyName>MSBuild.SonarQube.Internal.PostProcess</AssemblyName>
+    <RootNamespace>SonarQube.TeamBuild.PostProcessor</RootNamespace>
+    <DebugType>Full</DebugType>
+    <CodeAnalysisRuleSet>SonarQube.Scanner.End.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SonarAnalyzer.CSharp" Version="6.7.0.4267" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SonarQube.Common\SonarQube.Common.csproj" />
+    <ProjectReference Include="..\SonarQube.TeamBuild.Integration\SonarQube.TeamBuild.Integration.csproj" />
+    <ProjectReference Include="..\SonarScanner.Shim\SonarScanner.Shim.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="License.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\AssemblyInfo.Shared.cs">
+      <Link>Properties\AssemblyInfo.Shared.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="License.txt" />
```

Comment:
It might be better to put this in the same item group as the _Remove="License.txt"_ above.